src/features/tags/components/TagManager/TagManagerController.tsx

```diff
@@ -40,7 +40,6 @@ export const TagManagerController: React.FunctionComponent<
   onUnassignTag,
 }) => {
   const [addTagButton, setAddTagButton] = useState<HTMLElement | null>(null);
-
```

Comment:
This is a meaningless change, that still made it into Git. Making changes like this is no problem (as long as it's just affecting empty lines, like here) but it's indicating to me that maybe your workflow for deciding what to commit and what not to commit can be improved, to make sure that you don't accidentally commit changes that you are not intending.